contracts/secret-storage/SecretManager.sol

```diff
@@ -530,23 +530,33 @@ contract SecretManager is
         SECRET_STORE.slashEnclave(_enclaveAddress, sizeLimit, STAKING_PAYMENT_POOL);
     }
 
-    /// @dev It removes the dead store payment from the usdc deposit of the secret.
-    function _getStoreUsdcPayment(
+    /**
+     * @dev It updates usdc deposit based on the payment to the selected store.
+     *      It is executed when - 
+     *      (1) alive check is submitted for terminated secretId
+     *      (2) dead check
+     *      (3) remove secret is called post termination
+     */
+    function _updateUsdcDepositPostPayment(
         uint256 _secretId,
         address _enclaveAddress,
         uint256 _enclaveIndex
-    ) internal view returns (uint256) {
+    ) internal {
         uint256 lastAliveTimestamp = SECRET_STORE.getSecretStoreLastAliveTimestamp(_enclaveAddress);
```

Comment:
can we update this variable name to `latestAliveTimestamp`?

Reply:
just for this function?

Reply:
yeah